library/std/src/os/android/net.rs

```diff
@@ -0,0 +1,5 @@
+//! Linux and Android-specific definitions for socket options.
+
+#![unstable(feature = "tcp_quickack", issue = "96256")]
+
```

Comment:
superfluous blank line

Reply:
I saw this blank line appearing in more files, so I took it as standard. I removed line 4 in all places